shared/metadata_util.go

```diff
@@ -14,17 +14,34 @@ import (
 	"strings"
 )
 
-const metadataArchiveURL = "https://api.github.com/repos/web-platform-tests/wpt-metadata/tarball"
+// metadataMapCached is the local cache of Metadata.
+var metadataMapCached map[string][]byte = nil
```

Comment:
We not only need this in-memory cache, but more importantly need to put it into memcache for use by webapp.